tutorials/getstarted.qmd

```diff
@@ -1,3 +1,312 @@
+maybe a bit “dull” point on this Lie group is the identity matrix `e`.
+
+```{julia}
+SO3 = SpecialOrthogonalGroup(3)
+g = [cos(π/3) -sin(π/3) 0.0; sin(π/3) cos(π/3) 0.0; 0.0 0.0 1.0]* [cos(π/4) 0.0 -sin(π/4); 0.0 1.0 0.0; sin(π/4) 0.0 cos(π/4)]
+e = Matrix{Float64}(I,3,3)
+g
+```
+The element `g` can be seen as a rotation by $π/3$ in the x-y-plane combines with a rotation by $π/4$ in the x-z plane.
+
+The tangent space $T_e𝒢$ plays a special role and is called the [`LieAlgebra`](@ref) $𝔤$.
+
+Similar to the ``[Riemannian exponential map](@extref `Base.exp-Tuple{AbstractManifold, Any, Any}`)``{=commonmark}
+The exponential function $\exp_𝒢: 𝔤 → 𝒢$ maps Lie algebra tangent vectors $X ∈ T_e𝒢$
+to a point on the Lie group. This is implemented in [`exp(G::LieGroup, X)`](@ref). It's inverse
+is the Lie group logarithmic function [`log(G::LieGroup, g)`](@ref).
+
+```{julia}
+X = log(SO3,g)
+is_point(LieAlgebra(SO3), X)
+```
+
+```{julia}
+exp(SO3,X)
+```
+
+The term “exponential” has at least three different meanings throughout Lie groups and Riemannian manifolds.
+To be precise, we call the just introduced one “exponential function”, since it often agrees with the [exponential](https://en.wikipedia.org/wiki/Exponential_function).
+
+Taking a closed look at `X` we see
+
+```{julia}
+X
+```
+
+an example that all elements from the Lie algebra are skew symmetric matrices.
+This allows for one way to see that we have three degrees of freedom, cf
+
+```{julia}
+manifold_dimension(SO3)
+```
+
+As the Lie algebra was introduced via a tangent space, we also know that it is a vector space.
+To “turn” the `X` into a vector with three elements, we need a basis of the tangent space
+to decompose `X` into and “read off” its components.
+While in general this can be done using the [bases of a tangent space](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/bases/) from [`ManifoldsBase.jl`](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/),
+a very specific one is [`vee(::LieAlgebra, X)`](@ref)
+
+```{julia}
+c = vee(LieAlgebra(SO3), X)
+```
+
+and its inverse, [`hat(G::LieAlgebra, c)`](@ref)
+
+```{julia}
+X_ = hat(LieAlgebra(SO3),c)
+```
+
+## Representing tangent vectors
+
+Consider the function of left group composition $λ_g(h) = g∘h$.
+Its differential (or push forward) $Dλ_g(e)$ maps from $T_e𝒢$ to $T_g𝒢$
+and is a linear bijection between both spaces.
+Its adjoint differential (or pullback) $D^*λ_g(e)$ maps back from $T_g𝒢$ to $T_e𝒢$.
+
+This allows to introduce an inner product on all tangent spaces, that smoothly varies with
+the base point, we obtain the left-invariant metric
+
+```math
+⟨X,Y⟩_g = ⟨D^*λ_g(e)[X],D^*λ_g(e)⟩_e,
+```
+
+since it is invariant if you use the differential of the left composition to identify tangent vectors.
+We can even “spare” the pullbacks in this definition, when spring tangent vectors as their representants in the Lie algebra.
+This is the default in `LieGroups.jl`.
```

Comment:
Maybe it would be a good idea to add that this metric in general is not right-invariant, except for compact Lie groups and their direct products with `Translations`.

Reply:
Sure, maybe a bit of a technical note, since one has to say before also that right invariant can be build in a similar way. Will add that.

Reply:
Sure. The most general statement of this kind that I know of is in
```julia
@article{LatifiToomanian:2013,
	title = {On the existence of bi-invariant {Finsler} metrics on {Lie} groups},
	volume = {7},
	issn = {2251-7456},
	doi = {10.1186/2251-7456-7-37},
	number = {1},
	journal = {Mathematical Sciences},
	author = {Latifi, Dariush and Toomanian, Megerdich},
	month = aug,
	year = {2013},
	pages = {37},
}
```
where they show that it's true not only for Riemannian metrics but also Finsler ones.